StacksAndQueues/LargestRectangleInHistogram.cpp

```diff
@@ -1 +1,35 @@
+int Solution::largestRectangleArea(vector<int> &hist) {
+    stack<int> s; 
+  
+    int max_area = 0; // Initalize max area 
+    int tp;  // To store top of stack 
+    int area_with_top; // To store area with top bar 
+    int i = 0; 
+    while (i < hist.size()) 
+    { 
+        if (s.empty() || hist[s.top()] <= hist[i]) 
+            s.push(i++); 
```

Comment:
```suggestion
            s.push(++i); 
```

Reply:
Sir, with this change the first bar will not be considered for the answer.

Reply:
Hi Nishta, thanks for your response. My apologies, I didn't realize I had published my comments, I was reviewing the change intermittently at work. 

---
You're right, discard this suggestion in that case. But as a general rule of thumb always prefer pre-increment operator instead of post-increment to avoid the extra local copy. :)